crates/uv-resolver/src/lock/target.rs

```diff
@@ -171,30 +218,54 @@ impl<'env> InstallTarget<'env> {
                 })?;
 
             if dev.prod() {
-                // Add the base package.
-                queue.push_back((root, None));
+                // Add the workspace package to the graph.
+                if let Entry::Vacant(entry) = inverse.entry(&dist.id) {
+                    entry.insert(petgraph.add_node(node!(dist)));
+                }
+
+                // Add an edge from the root.
+                let index = inverse[&dist.id];
+                petgraph.add_edge(root, index, Edge::Prod(MarkerTree::TRUE));
```

Comment:
I really that we don't have orphan nodes anymore, but aren't those a bit different in that they aren't mandatory, but depend on the user selection? (This may just be a documentation issue)